### DISCUSSION TITLE
mmrotate 与 mmcv和mmdetection的版本兼容问题

### DISCUSSION THREAD

**Discussion by foeverai** [open-mmlab/mmdetection#11640] [Q&A]
我的电脑上安装的是detection3.2版本的，但是我并没有在mmrotate中找到合适的版本，请问除了重新安装就没有其他的方法吗

**Comment by Mariiiiooooo** [comment]
请问解决了吗

**Reply by Dong1C** [reply]
降mmdetection版本到2.8

**Comment by Mariiiiooooo** [comment]
感谢感谢！

---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2025年02月01日 10:15 |
| 收件人 | open-mmlab/mmdetection ***@***.***> |
| 抄送人 | Mariiiiooooo ***@***.***>,
Comment ***@***.***> |
| 主题 | Re: [open-mmlab/mmdetection] mmrotate 与 mmcv和mmdetection的版本兼容问题 (Discussion #11640) |

降mmdetection版本到2.8
